frontend/server/src/Controllers/Tag.php

```diff
@@ -72,13 +72,15 @@ public static function getPublicTags() {
      * @return list<array{alias: string}>
      */
     public static function getFrequentTagsByLevel(
-        string $problemLevel
+        string $problemLevel,
+        int $rows
     ): array {
         return \OmegaUp\Cache::getFromCacheOrSet(
             \OmegaUp\Cache::TAGS_LIST,
             "level-{$problemLevel}",
```

Comment:
```suggestion
            "level-{$problemLevel}-{$rows}",
```

si no, una invocación con `$rows=1` va a traer el mismo resultado que una invocación con `$row=15` si esa se hace primero!

en general cuando se manda llamar `getFromCacheOrSet()`, el nombre de la llave debe contener todos los parámetros de la función de alguna manera.